docs/_posts/examples/3400-01-14-toggle-interaction-handlers.html

```diff
@@ -0,0 +1,95 @@
+---
+layout: example
+category: example
+title: Toggle interactions
+description: Enable or disable UI handlers on a map.
+---
+<style>
+    .listing-group {
+        font: 12px/20px 'Helvetica Neue', Arial, Helvetica, sans-serif;
+        font-weight: 600;
+        position: absolute;
+        top: 10px;
+        right: 10px;
+        z-index: 1;
+        border-radius: 3px;
+        max-width: 20%;
+        color: #fff;
+    }
+
+    .listing-group input[type=checkbox]:first-child + label {
+        border-radius: 3px 3px 0 0;
+    }
+
+    .listing-group label:last-child {
+        border-radius: 0 0 3px 3px;
+        border: none;
+    }
+
+    .listing-group input[type=checkbox] {
+        display: none;
+    }
+
+    .listing-group input[type=checkbox] + label {
+        background-color: #3386c0;
+        display: block;
+        cursor: pointer;
+        padding: 10px;
+        border-bottom: 1px solid rgba(0, 0, 0, 0.25);
+    }
+
+    .listing-group input[type=checkbox] + label {
+        background-color: #3386c0;
+        text-transform: capitalize;
+    }
+
+    .listing-group input[type=checkbox] + label:hover,
+    .listing-group input[type=checkbox]:checked + label {
+        background-color: #4ea0da;
+    }
+
+    .listing-group input[type=checkbox]:checked + label:before {
+        content: '✔';
+        margin-right: 5px;
+    }
+</style>
+<div id='map'></div>
+<nav id='listing-group' class='listing-group'>
+    <input type='checkbox' id='scrollZoom' checked='checked'>
+    <label for='scrollZoom'>Scroll zoom</label>
+    <input type='checkbox' id='boxZoom' checked='checked'>
+    <label for='boxZoom'>Box zoom</label>
+    <input type='checkbox' id='dragRotate' checked='checked'>
+    <label for='dragRotate'>Drag rotate</label>
+    <input type='checkbox' id='dragPan' checked='checked'>
+    <label for='dragPan'>Drag pan</label>
+    <input type='checkbox' id='keyboard' checked='checked'>
+    <label for='keyboard'>Keyboard</label>
+    <input type='checkbox' id='doubleClickZoom' checked='checked'>
+    <label for='doubleClickZoom'>Double click zoom</label>
+    <input type='checkbox' id='touchZoomRotate' checked='checked'>
+    <label for='touchZoomRotate'>Touch zoom rotate</label>
+</nav>
+
+<script>
+var listingGroup = document.getElementById('listing-group');
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v8',
+    center: [-77.04, 38.907],
+    zoom: 11.15
+});
+
+map.on('load', function() {
+    Array.prototype.forEach.call(listingGroup.querySelectorAll('input'), function(el) {
```

Comment:
Instead of binding multiple event listeners, do `document.getElementById('listing-group').addEventListener(...)` and rely on event bubbling.